examples/testimonials/modules/testimonial/views/latest.html

```diff
@@ -0,0 +1,15 @@
+<div class="container mx-auto mx-2">
```

Comment:
While I'm not sure how many people would actually use this template, maybe it is worth cherry-picking the necessary Tailwind CSS and packaging it up for delivery in the `ui/src/index.scss` file.